src/app/messages/pl/messages.json

```diff
@@ -220,7 +220,7 @@
   "i18n:toolbar:highlights:popup:heading": "Moje notatki i zakreślenia",
   "i18n:toolbar:highlights:popup:heading:no-highlights": "Brak wyników",
   "i18n:toolbar:highlights:popup:heading:no-highlights-tip": "Wybierz inny <strong>rozdział albo kolor</strong>, żeby zobaczyć inne wyniki.",
-  "i18n:toolbar:highlights:popup:help-info": "Potrzebujesz pomocy? Przejdź do <a data-testid='support-link' data-analytics-label='support-link' target=\"_blank\" href=\"https://help.openstax.org/s/article/Using-the-highlighting-and-note-taking-feature-on-mobile-or-tablet?search=highlighting\">Support Center</a>",
+  "i18n:toolbar:highlights:popup:help-info": "Notatek i wyróżnień nie można edytować na urządzeniach mobilnych. <a data-testid='support-link' data-analytics-label='support-link' target=\"_blank\" href=\"https://help.openstax.org/s/article/Using-the-highlighting-and-note-taking-feature-on-mobile-or-tablet?search=highlighting\">Przeczytaj więcej</a>",
```

Comment:
@ewaforman Please check this translation of "Notes and highlights cannot be edited on mobile devices. Read more"